CS483/CS483/Kartaclysm/Components/ComponentTrack.cpp

```diff
@@ -289,7 +297,61 @@ namespace Kartaclysm
 		if (bRaceStandingsUpdate)
 		{
 			TriggerRaceStandingsUpdateEvent();
+			m_bHumanPositionsDirty = true;
+		}
+	}
+
+	void ComponentTrack::UpdateHumanPositions()
```

Comment:
Does this need to be separate from `ComponentTrack::UpdateRacerPositions()`? Would save us from looping through each racer yet again.